libs/SalesforceSDK/res/values/strings.xml

```diff
@@ -12,5 +12,5 @@
     <string name="oauth_display_type">touch</string>
 
     <!-- Default API version used by the SDK and sample apps -->
-    <string name="api_version">v63.0</string>
+    <string name="api_version">v64.0</string>
```

Comment:
It's okay for testing but we probably can't merge that into dev until v64.0 has rolled out to our test orgs instances at least.